mgmtfn/k8splugin/kubeClient.go

```diff
@@ -86,7 +86,7 @@ func (p *podInfo) setDefaults(ns, name string) {
 	p.name = name
 	p.labels["tenant"] = "default"
```

Comment:
Do we want to name the labels as 'io.contiv.tenant', and similarly 'io.contiv.network', etc.?
This could avoid potential conflicts...